app/routes/jokes[.]rss.tsx

```diff
@@ -35,9 +39,9 @@ export let loader: LoaderFunction = async ({ request }) => {
           .map((joke) =>
             `
             <item>
-              <title>${joke.name}</title>
-              <description>A funny joke called ${joke.name}</description>
-              <author>${joke.jokester.username}</author>
+              <title><![CDATA[${escapeCdata(joke.name)}]]></title>
+              <description><![CDATA[A funny joke called ${escapeCdata(joke.name)}]]></description>
```

Comment:
Note that most RSS readers will interpret the HTML in this tag, this is probably unwanted.